tetris.py

```diff
@@ -127,10 +127,27 @@ def new_block(self):
             "I": [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)],
         }
```

Comment:
I wish I could do a really long multiline suggestion, but I can't:
- Delete `self.blocks`
- Create a new attribute, `self.shape_names` maybe, with the value `["L", "L_rev", "O", "E", "Z", "Z_rev"]`
- Replace `list(self.blocks.keys()) with `self.shape_names` everywhere

Currently there same lists are copy/pasted in `self.blocks` and `get_current_block`. This allows them to be only in `get_current_block`.

Also, you could use a global variable instead of `self.shape_names`. The list of shape names might be useful in GUI code too.